src/containers/Routes.tsx

```diff
@@ -21,8 +22,8 @@ function MainRoutes() {
         <Routes>
           <Route path="/" element={<Home />} />
           <Route path="/register-organization" element={<OrgRegister />} />
-          <Route path="/org-login" element={<Orglogin />} />
-          <Route path="/login-admin" element={<Adminlogin />} />
+          <Route path="/org-login" element={<ProtectedRoutes redirectTo="/"><Orglogin /></ProtectedRoutes>}  />
+          <Route path="/login-admin" element={<ProtectedRoutes redirectTo="/"><Adminlogin /></ProtectedRoutes>} />
```

Comment:
Same here, do not just redirect. They need to know why they can not access the page, a simple redirect is terrible for the user experience